devel/libuv/Portfile

```
@@ -75,8 +75,10 @@ configure.args  --disable-silent-rules
 
 platform darwin {
     if { ${os.major} == 8 } {
-        # Tiger has no libutil
-        patchfiles-append patch-no-libutil-on-Tiger.diff
+        post-patch {
+            # Tiger has no libutil
+            reinplace -E "s|(.*DARWIN_TRUE.*-lutil.*)|# \\1|" Makefile.in
```

@barracuda156 Do you know if this is sufficient as compared with the patch? The patch originally commented out DragonFlyBSD's, FreeBSD's, etc as well which seemed unnecessary for Tiger to work.

If it is necessary then the regex could instead be:
```suggestion
            reinplace -E "s|(.*-lutil.*)|# \\1|" Makefile.in
```
which would yield a similar result to the patch.

(Extra context: I changed this to be a `reinplace` instead because this file is dynamically created by automake so will more likely apply in the future)

---

@casr No idea but I can test that if needed. I think that MacPorts generally tends to prefer patches over reinplacing (patches are safer but more involving to maintain).